src/pages/api/featuredItem.ts

```diff
@@ -2,8 +2,8 @@ import {
   getNumDigitizedItems,
   getFeaturedImage,
   imageURL,
-} from "@/utils/utils";
-import appConfig from "appConfig";
+} from "../../app/utils/utils";
```

Comment:
Is this file still being used or is it the one in `src/app/pages/api/feedback.ts`? If you don't need both, delete the unused API directory.

Reply:
This is the one is still being used– removed`src/app/pages/api`